types/react-router-redux/index.d.ts

```diff
@@ -25,15 +25,15 @@ export interface ConnectedRouterProps<State> {
 }
 export class ConnectedRouter<State> extends React.Component<ConnectedRouterProps<State>> {}
 
-export const LOCATION_CHANGE: string;
+export const LOCATION_CHANGE: '@@router/LOCATION_CHANGE';
```

Comment:
I think it's better to have `export const LOCATION_CHANGE = '@@router/LOCATION_CHANGE'`. typescript will infer `LOCATION_CHANGE` to have the literal type.